pocs/tests/conftest.py

```diff
@@ -15,14 +15,22 @@
 _one_time_config = None
 
 
+@pytest.fixture(scope='module')
+def images_dir(tmpdir_factory):
+    directory = tmpdir_factory.mktemp('images')
+    return str(directory)
+
+
 @pytest.fixture(scope='function')
-def config():
+def config(images_dir):
     pocs.base.reset_global_config()
 
     global _one_time_config
     if not _one_time_config:
         _one_time_config = load_config(ignore_local=True, simulator=['all'])
         _one_time_config['db']['name'] = 'panoptes_testing'
+        _one_time_config['directories']['images'] = images_dir
```

Comment:
Will this line execute often enough? Or should it indented less?

Reply:
Would that be true of the db as well? Or because that is a global is it fine anyway?